introduction.md

```diff
@@ -5,3 +5,6 @@ $ git push
 $ echo "On my way to becoming a GitHub Master. I love Ruby and JavaScript." > introduction.md
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?